cds/modules/deposit/mappings/os-v2/deposits/records/videos/video/video-v1.0.0.json

```diff
@@ -433,7 +433,7 @@
         }
       },
       "collections": {
-        "type": "text"
+        "type": "keyword"
```

Comment:
Are we sure about this change, with this it means that we couldn't search for all lectures, or any parent, only for the child collections? What's the motivation behind this change?

Reply:
Ok let's discuss with him when he is back but I think all of the values should be searchable, there are different ways to achieve this but we can discuss once we confirmed it.